scripts/system_performance/system_performance_queries.py

```diff
@@ -0,0 +1,159 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+
+"""
+The purpose of this script is for performing MySQL queries to monitor system state
+performance and health.
+"""
+from __future__ import print_function
+import itertools
+import logging
+from utils.clients.database_client import DatabaseClient
+from utils.clients.connection_exception import ConnectionException
+
+
+class DatabaseMonitorChecks(object):
+    """Class to query system performance"""
+    table = "reduction_viewer_reductionrun"
+
+    # Establishing a connection with Database using Database Client
+    def __init__(self):
+        self.database = DatabaseClient()
+        try:
+            self.connection = self.database.connect()
+        except ConnectionException:
+            raise ConnectionException('database')
+
+    def query_log_and_execute(self, constructed_query):
+        """Logs and executes all queries ran in script"""
+        logging.info('SQL QUERY: %s', constructed_query)
+        print(constructed_query)
+        return self.connection.execute(constructed_query).fetchall()
+
+    def instruments_list(self):
+        """Retrieve current list of instruments"""
```

Comment:
Can be more verbose - """ Retrieve current list of instruments **from database**
 """